rflint/rflint.py

```diff
@@ -155,6 +156,10 @@ def _process_file(self, filename):
         self._print_filename = filename if self.args.print_filenames else None
 
         robot_file = RobotFactory(filename)
+
+        for skip_line in robot_file.skip_lines:
+            self.noqa.append({"path": robot_file.path, "line": skip_line})
```

Comment:
You can make noqa also dict to faster look up. It could look like:

{
   '/some/path/file1': (123),
   'other/path/file.robot': (150, 140)
}
Then:
```code
if filename in self.noqa and linenumber in self.noqa[filename]:
    self.noqa.append({"path": robot_file.path, "line": skip_line})
        return
```